baseframe/__init__.py

```diff
@@ -321,6 +323,27 @@ def get_timezone():
     return current_app.config.get('tz') or UTC
 
 
+def localized_country_list():
+    # Returns a localized list of country names and the ISO3166 code
+    return _localized_country_list_inner(get_locale())
+
+
+@cache.memoize(timeout=86400)
+def _localized_country_list_inner(locale):
+    if locale == 'en':
+        countries = [(country.name, country.alpha_2) for country in pycountry.countries]
+    else:
+        pycountry_locale = gettext.translation('iso3166-1', pycountry.LOCALES_DIR, languages=[locale])
+        countries = []
+        for country in pycountry.countries:
+            name = pycountry_locale.gettext(country.name)
+            if six.PY2:
+                name = name.decode('utf-8')
+            countries.append((name, country.alpha_2))
```

Comment:
This version will be significantly faster because it (a) uses a list comprehension and (b) moves the `if` condition out of the loop:

```python
if six.PY2:
    countries = [(pycountry_locale.gettext(country.name).decode('utf-8'), country.alpha_2) for country in pycountry.countries]
else:
    countries = [(pycountry_locale.gettext(country.name), country.alpha_2) for country in pycountry.countries]
```